src/models/message.model.js

```diff
@@ -0,0 +1,29 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db');
+const { User } = require('./user.model');
+const { Room } = require('./room.model');
+
+const Message = sequelize.define(
+  'message',
+  {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
+      allowNull: false,
+    },
+  },
+  {
+    updatedAt: false,
+  },
```

Comment:
The `updatedAt: false` option should be placed inside the `options` object of the model definition. Ensure it is correctly nested to disable the `updatedAt` timestamp.